src/zaff_tools.msag.xml

```diff
@@ -20,12 +20,6 @@
      <MSGNR>102</MSGNR>
      <TEXT>The JSON type &amp;1 is not supported by the XSLT writer</TEXT>
     </T100>
-    <T100>
-     <SPRSL>E</SPRSL>
-     <ARBGB>ZAFF_TOOLS</ARBGB>
-     <MSGNR>105</MSGNR>
-     <TEXT>Component &amp;1 of constant &amp;2 in ABAP Doc link doesn&apos;t exist</TEXT>
-    </T100>
     <T100>
      <SPRSL>E</SPRSL>
      <ARBGB>ZAFF_TOOLS</ARBGB>
```

Comment:
</T100>

Reply:
I'm sorry, what change do you suggest?

Reply:
</T100> at line 26 to close it.maybe

Reply:
In my opinion, this message class is correct, so I would resolve this conservation und merge this pull request. Feel free to raise an issue if you see any further problem.